flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/internal/TableEnvImpl.scala

```diff
@@ -385,6 +389,67 @@ abstract class TableEnvImpl(
     planner.getCompletionHints(statement, position)
   }
 
+  override def sql(statement: String): Unit = {
```

Comment:
why the behavior of `sql` method in this class is different from the behavior of `sql` method in `TableEnvironmentImpl` ?

Reply:
I also have the same concern. It seems that it doesn't support multiple statements in this env. I think we should keep the same behavior.